src/commcare_cloud/ansible/roles/formplayer/templates/monit.formplayer.j2

```diff
@@ -0,0 +1,5 @@
+ check file formplayer with path {{ formplayer_logging_dir }}/formplayer-spring.log
+  group formplayer
+  start program = "supervisorctl start {{ project }}-{{ deploy_env }}-formsplayer-spring"
+  stop program = "supervisorctl stop {{ project }}-{{ deploy_env }}-formsplayer-spring"
+  if content = "java.lang.OutOfMemoryError" then restart
```

Comment:
we might want to make this only for the gc overhead one, not all OOMs, because I think it recovers from individual thread OOMs relative gracefully.

Reply:
Also, and this really is the most annoying thing for me to ask, but is there any user input that they can put here that makes this literal string show up in the log? If so we're sort of giving people a button they can press to restart a process on our server. I don't want to block the PR on this, but if there's a more complete way to match against a line such that a user can't make that line show up in the file at will, then it'd be worth doing that. For example if we know that the line will always start with a like a timestamp in brackets then a space, then the specific error with no text in between, then you can make a regular expression match it including a ^ start of line match that would make it much harder to do that way. Really annoying comment, I know

Reply:
Fortunately in the logs, there is a newline prior to output containing the OOM so added this regex:
`"^Caused by: java.lang.OutOfMemoryError: GC overhead limit exceeded"`

Reply:
This made me think of another possible issue. I assume end users are unable to enter a string with newlines that end up in the logs? If not, that seems like a problem here. However, if that is an issue I propose that we deal with it in a follow-up PR rather than dragging this one out longer.